File: help/pt_pt/exercicios_da_ficha_3/Ficha03_002/Ficha03_002.java
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_002 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Valor de X\t: " );
        double x = Double.valueOf(keyb.nextLine());
        System.out.print("\nRaiz \t\t= " + Math.sqrt( x ) );
        System.out.print("\nPotência \t= " + Math.pow( x , 0.5 ) );
        System.out.print("\nOperador \t= " + Math.pow( x , 0.5 ) );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 05 20:25

File: help/pt_pt/exercicios_da_ficha_3/Ficha03_002/Ficha03_002.php
<?php
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Valor de X    : " ;
    $x = floatval(isset($_POST["x"])? $_POST["x"] : isset($_GET["x"]) ? $_GET["x"] : 0);
    
    echo $x . "<br/>";// print input 
    echo  "\nRaiz         = " . ( sqrt( $x )  ) ;
    echo  "\nPotência     = " . ( pow( $x , 0.5 )  ) ;
    echo  "\nOperador     = " . ( $x ^ 0.5  ) ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	quarta, 2022 out 05 20:25

?>